res/http-export/configuration.toml

```diff
@@ -47,8 +47,8 @@ Type = "mongodb"
 Host = "localhost"
 Port = 27017
 Timeout = "30s"
-Username = ""
-Password = ""
+Username = "appservice"
+Password = "appservice"
```

Comment:
Password should be `password`

https://github.com/edgexfoundry/docker-edgex-mongo/blob/master/cmd/res/configuration.toml#L68

Reply:
oh,  resolved